tests/test_websockets.py

```diff
@@ -315,6 +315,20 @@ async def asgi(receive, send):
         assert websocket.extra_headers == [(b"additional", b"header")]
 
 
+def test_no_additional_headers(test_client_factory):
+    def app(scope):
+        async def asgi(receive, send):
+            websocket = WebSocket(scope, receive=receive, send=send)
+            await websocket.accept(headers=None)
```

Comment:
I'm not sure about this, because the `headers` parameter doesn't accept `None` as argument. The annotation there is `typing.Iterable[typing.Tuple[bytes, bytes]]`. We can do this instead:

```suggestion
            await websocket.accept()
```

Reply:
I rather agree. I was worried that it would not be clear, what exactly we are testing.